hazelcast-client/src/main/java/com/hazelcast/client/impl/clientside/HazelcastClientInstanceImpl.java

```diff
@@ -542,6 +544,16 @@ private ClientExecutionServiceImpl initExecutionService() {
                 config.getClassLoader(), properties, config.getExecutorPoolSize(), loggingService);
     }
 
+    private void logGroupPasswordInfo() {
```

Comment:
method name may be changed to checkAndLogGroupPasswordInfo, since it may not log based on the if condition.